src/commands/appdistribution-distribute.ts

```diff
@@ -0,0 +1,109 @@
+import * as fs from "fs-extra";
+
+import * as Command from "../command";
+import * as utils from "../utils";
+import * as requireAuth from "../requireAuth";
+import * as req from "../appdistribution/requests";
```

Comment:
Maybe just `import { AppDistributionClient } from "../appdistribution/client"` (also, can you rename `requests.ts` to `client.ts`)?

Reply:
Done.